node/src/main/scala/coop/rchain/node/main.scala

```diff
@@ -208,6 +188,37 @@ object Main {
       def toEffect: Effect[A] = t.liftM[CommErrT]
     }
 
+    val net = new UnicastNetwork(src, Some(p2p.Network))
```

Comment:
No need to `new`, yah?

Reply:
well, this should not really be case class, but i will remove `new` for now

Reply:
yeah, one or the other, i guess

Reply:
I've ended up changing `UnicastNetwork` to be normal class, it never should be case class in the first place.